src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetSecureRandomCommand.cs

```diff
@@ -0,0 +1,716 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+using System.Numerics;
+using System.Security.Cryptography;
+using System.Threading;
+
+using Debug = System.Management.Automation.Diagnostics;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// This class implements Get-SecureRandom cmdlet.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "SecureRandom", DefaultParameterSetName = GetSecureRandomCommand.RandomNumberParameterSet,
+        HelpUri = "https://go.microsoft.com/fwlink/?LinkID=", RemotingCapability = RemotingCapability.None)]
+    [OutputType(typeof(int), typeof(long), typeof(double))]
+    public class GetSecureRandomCommand : PSCmdlet
```

Comment:
```suggestion
    public sealed class GetSecureRandomCommand : PSCmdlet
```

Reply:
`GetRandom` derives from this, but I can make that one sealed

Reply:
I think it should be _sealed_ and both cmdlets should have the same _internal_ ancestor.

Reply:
That's a good suggestion.  I'll make that change.